modules/ocf_www/files/build-vhosts

```diff
@@ -161,6 +176,13 @@ def build_config(src_vhosts, template, dev_config=False):
             bind_type = 'docroot'
         bind_dest = vhost[bind_type]
 
+        # Match and process supported flags
+        additional_rules = {}
+        for flag in vhost['flags']:
+            ws_match = re.match(r'ws\=(.+)', flag)
+            if ws_match:
+                additional_rules.setdefault('ws', []).append(ws_match.group(1))
```

Comment:
Heh, this is very clever. Might be more readable on two lines though

Reply:
It didn't look nice when I tried the two line thing so I just left it